.github/workflows/flake8.yaml

```
@@ -0,0 +1,20 @@
+---
+name: Flake8
+on: pull_request
+jobs:
+  flake8:
+    name: Check code with Flake8
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.10
+      - name: Setup environment
+        run: |
+          python -m pip install --upgrade pip
+          pip install flake8
+      - name: Run Flake8
+        run: flake8 . --ignore=E266 --max-line-length=120 --count
```

Gave a run to flake8. Encountered 256 errors and even more with pylint. The CI for running the unit tests is non-negotiable but some of the lint  are. I would take out stuff like E261,E266, E302. Let's use the process to help getting to the goal and  not the other way around.

```suggestion
        run: flake8 . --ignore=E266,E261, E265, E302 --max-line-length=120 --count
```
Also saw we have already some pylint disable that will be needed to be deleted  in some lines.

---

Well, these are following [PEP8](https://peps.python.org/pep-0008/#block-comments) styling guide. I do think E302 is required for readability